franklin/config/settings/base.py

```diff
@@ -106,8 +106,8 @@
     'social.pipeline.user.get_username',
     'social.pipeline.user.create_user',
     'social.pipeline.social_auth.associate_user',
-    'core.views.save_oauth',
     'social.pipeline.social_auth.load_extra_data',
+    'core.views.save_oauth',
```

Comment:
moving this 1 step later fixes a bug with our current social auth implementation